web/src/features/dashboard/components/UserChart.tsx

```diff
@@ -34,7 +36,9 @@ export const UserChart = ({
   const user = api.dashboard.chart.useQuery(
     {
       projectId,
-      from: "traces_observationsview",
+      from: env.NEXT_PUBLIC_HAS_BACKFILLED_OBSERVATION_COST
```

Comment:
Can you add it to the env.production file?
Maybe we should also rename this to NEXT_PUBLIC_LANGFUSE_3_MIGRATION_DONE or similar

Reply:
fyi, we should try to not use NEXT_PUBLIC envs as they get flattened at build-time, thus, they are always null or static for our prebuilt docker images. Can we do this via a regular env that gets used server-side or NEXT_PUBLIC_CLOUD_ENV which only gets used in our cloud deployment and is passed at build-time in or cloud build pipeline?

Reply:
As discussed, lets take NEXT_PUBLIC_LANGFUSE_CLOUD_REGION here.